lib/open_api/docs/component_collection.rb

```diff
@@ -13,7 +13,7 @@ def [](name)
       end
 
       def load_definition(name)
-        raw_definition = @doc.content[@category][name]
+        raw_definition = @doc.content.fetch_path(@category.split("/"))[name]
```

Comment:
If you're using fetch_path, you can put the name in the fetch as well

```ruby
@doc.content.fetch_path(*@category.split("/"), name)
```

Reply:
👍